packages/style-engine/src/index.ts

```diff
@@ -62,3 +65,20 @@ export function getCSSRules(
 
 	return rules;
 }
+
+/**
+ * Returns an array of classnames.
+ *
+ * @param  style Style object.
+ *
+ * @return string[] An array of classnames.
+ */
+export function getClassnames( style: Style ): string[] {
+	const classNames: string[] = [];
+	styleDefinitions.forEach( ( definition: StyleDefinition ) => {
+		if ( typeof definition.getClassNames === 'function' ) {
+			classNames.push( ...definition.getClassNames( style ) );
+		}
+	} );
+	return uniq( classNames );
```

Comment:
I know we're already using lodash here (and might not be able to remove it in the shorter-term), but instead of `uniq` I was wondering if we could use a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) for `classNames`, and then return it as an array via `return Array.from( classNames )`?

Reply:
That would work! Thanks for the tip.